packages/react/src/components/iconBadge/IconBadge.tsx

```diff
@@ -51,16 +49,12 @@ const TypeColorMapping: Record<IconBadgeType, string> = {
     [IconBadgeType.Major]: 'mod-major',
 };
 
-export const IconBadge: FunctionComponent<IconBadgeProps> = ({
-    svgName,
-    type,
-    size = IconBadgeSize.Medium,
-    svgClass,
-    className,
-}) => (
-    <Svg
-        className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}
-        svgName={svgName}
-        svgClass={classNames('icon align-middle', SizeClassMapping[size], svgClass)}
-    />
-);
+export const IconBadge: FunctionComponent<IconBadgeProps> = ({icon, type, size = IconBadgeSize.Medium, className}) => {
+    const IconName = icon;
+
+    return (
+        <div className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}>
```

Comment:
The size mapping should be applied in the `height` prop of the icon I think. Adding a mod-24 to this div has no effect.

Reply:
mod-24 is for the dot, not the icon actually ^^
I was also wondering if we really need to have this flexibility on the size? I though those were getting deprecated soon? 🤔